src/System.Memory/src/System/Buffers/ReadOnlySequence.cs

```diff
@@ -394,7 +394,7 @@ public bool MoveNext()
 
         private enum SequenceType
         {
-            IMemoryList = 0x00,
+            Segment = 0x00,
```

Comment:
Should this be ReadOnlySequenceSegment instead of Segment?

Reply:
I would prefer something that shows that it's multiple segments, `MultiSegment`?